taxonomies/categories.txt

```diff
@@ -8370,11 +8365,14 @@ fr:Concentrés de jus de fruits, jus de fruits concentrés
 en:Apple juice concentrates, concentrated apple juices
 fr:Concentrés de jus de pommes, Jus de pommes concentrés
 
-<en:Squeezed lemon juices
 <en:Fruit juice concentrates
 en:Lemon juice concentrates, concentrated lemon juices
 fr:Concentrés de jus de citrons, Jus de citrons concentrés
 
+<en:Fruit juice concentrates
```

Comment:
You can remove this block because "concentrates, concentrated, juice, juices" in EN as well as "concentrés de", "jus de", "concentrés" in FR are in ingredients_processing.txt.

Reply:
But this is the category taxonomy

Reply:
correct! my mistake